src/view/com/util/post-embeds/index.tsx

```diff
@@ -138,13 +147,27 @@ export function PostEmbeds({
         alt: img.alt,
         aspectRatio: img.aspectRatio,
       }))
-      const _openLightbox = (index: number) => {
+      const _openLightbox = (
+        index: number,
+        thumbDims: MeasuredDimensions | null,
+      ) => {
         openLightbox({
           type: 'images',
           images: items,
           index,
+          thumbDims,
         })
       }
+      const onPress = (
+        index: number,
+        ref: AnimatedRef<React.Component<{}, {}, any>>,
+      ) => {
+        runOnUI(() => {
+          'worklet'
+          const dims = measure(ref)
+          runOnJS(_openLightbox)(index, dims)
```

Comment:
This adds a bit of nested thread hopping. Not sure if a concern. Worth checking if it got slower. Not sure if there's another way to get measurements (I only want to do them lazily when actually needed, and not always).